src/DataFrame/IO/Parquet/Binary.hs

```diff
@@ -30,6 +30,7 @@ word32ToLittleEndian w = map (\i -> fromIntegral (w `shiftR` i)) [0, 8, 16, 24]
 readUVarInt :: [Word8] -> (Word64, [Word8])
 readUVarInt xs = loop xs 0 0 0
   where
+    loop [] x _ _ = undefined
```

Comment:
This part is weird. I'm not quite sure what the intended behavior was even supposed to be. Fine to leave as undefined now but I'll have to rethink about this whole bit manipulation section

Reply:
I learned more about how thrift encodes ints and filled in the missing branch accordingly